data-structures.md

```diff
@@ -161,18 +157,14 @@ type Signature struct {
 
 ### `Type` Values
 
-| Key Type | Value |
-|-------|----------|
-|  Secp256k1 | 1 |
-| BLS12-381 ECDSA | 2 |
+| Key Type        | Value |
+|-----------------|-------|
+| Secp256k1       | `1`     |
+| BLS12-381 ECDSA | `2`     |
 
 ### Serialization
 
-`<uvarint(Type)><Data>`
-
-Note: As signatures should always be within wrapper types, length prefixing is not needed here.
-
-
+Uses FCS.
```

Comment:
hrm... allocating a code for signatures in FCS doesnt feel necessary to me, as signatures are never serialized independent of the 'SignedMessage' type.

Reply:
Shouldn't we just inline them then instead? I am worried that we will use them in different places, and then end up with them by themselves, when you eg traverse these structures using ipld